.vscode/launch.json

```diff
@@ -18,7 +18,7 @@
             ],
             "preLaunchTask": {
                 "type": "npm",
-                "script": "watch"
+                "script": "compile"
```

Comment:
I changed this because it was never starting that additional, special window in vscode that you need to test the extension. It would get stuck on `watch` instead and that is it, since this is prelaunch task, and since it never ended, extension never got launched.
So now you do need to do some manual work to pick up new changes, but at least it works. I don't understand how this `watch` here ever worked hm.